system-tests/lib/fixtures.ts

```diff
@@ -11,6 +11,21 @@ const serverRoot = _path.join(__dirname, '../../packages/server/')
 const projects = _path.join(root, 'projects')
 const cyTmpDir = _path.join(tempDir, 'cy-projects')
 
+const safeRemove = (path) => {
+  try {
+    fs.removeSync(path)
```

Comment:
Can we not keep this async? i.e. make safeRemove async and change this to await fs.remove?